IHP/Fetch.hs

```diff
@@ -55,7 +55,6 @@ instance (model ~ GetModelByTableName table, KnownSymbol table) => Fetchable (Qu
     fetch !queryBuilder = do
         let !(theQuery, theParameters) = queryBuilder
                 |> toSQL
-        logQuery theQuery theParameters
```

Comment:
Shouldn't this call the logger here?

Reply:
I found that all queries constructed with the query builder and called with fetch were being logged twice: once in the fetch functions, and once in `sqlQuery`. Since all the queries built are eventually dispatched to `sqlExec` or `sqlQuery` , which both log the query, the logs here aren't needed. I might be missing something but I'm pretty sure this was a bug before